src/js/edu-benefits/5490/config/form.js

```diff
@@ -73,7 +75,14 @@ const formConfig = {
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
-      pages: {}
+      pages: {
+        applicantInformation: applicantInformation([
+          'relativeFullName',
```

Comment:
I'm pretty sure these are the right ones, but yeah @goldenmeanie can verify.

My understanding is this form is for the dependent to apply for the transfer of benefits because the sponsor veteran is MIA, POW, or deceased and can't transfer the benefit. The applicant in this case is the relative, not the veteran.